### PR TITLE
Test: when non responding forwarder is used, wrong error message is reported.

### PR DESCRIPTION
**Pull request by amore17**
related : https://pagure.io/freeipa/issue/6989

### CODE REVIEW

**Comment by rcritten** [comment]
PEP-8 errors:
./ipatests/test_integration/test_installation.py:381:9: E265 block comment should start with '# '
./ipatests/test_integration/test_installation.py:383:9: E265 block comment should start with '# '
./ipatests/test_integration/test_installation.py:385:9: E265 block comment should start with '# '
./ipatests/test_integration/test_installation.py:396:20: E127 continuation line over-indented for visual indent
./ipatests/test_integration/test_installation.py:397:20: E127 continuation line over-indented for visual indent
./ipatests/test_integration/test_installation.py:398:80: E501 line too long (87 > 79 characters)

**Comment by pvoborni** [comment]
Also, I'd suggest running `make fastlint` before creating PR a fix all issues it reports. It will help with fixing issues caught by pycodestyle checks in Travis.

https://www.freeipa.org/page/Testing#Fast_test

Another thing: the PR needs to be rebased on clear master without the invalid commit, its revert and the merge commit.

File: ipatests/test_integration/test_installation.py [hunk 1]
```diff
@@ -362,3 +363,30 @@ def test_reserved_ip_as_forwarder(self):
...
+            ['mv', paths.RESOLV_CONF, '/tmp'])
+        content = ('search {domain}\nnameserver {master_ip}'
+                   .format(domain=master.domain.name, master_ip=master.ip))
+        master.put_file_contents(paths.RESOLV_CONF, content)
+        tasks.install_master(self.master, setup_dns=True)
+
+        rev_zone = master.ip.split('.')[2] + '.' + \
+                   master.ip.split('.')[1] + '.' + \
+                   master.ip.split('.')[0] + '.in-addr.arpa.'
+        msg = 'Reverse zone %s for IP address %s already exists', (rev_zone, master.ip)
```
Comment (flo-renaud) [inline ipatests/test_integration/test_installation.py]: This line should be:
```
msg = 'Reverse zone %s for IP address %s already exists' % (rev_zone, master.ip)
```

File: ipatests/test_integration/test_installation.py [hunk 2]
```diff
@@ -362,3 +363,30 @@ def test_reserved_ip_as_forwarder(self):
...
+        assert msg not in paths.IPASERVER_INSTALL_LOG
```
Comment (flo-renaud) [inline ipatests/test_integration/test_installation.py]: You need to use `result = tasks.install_master` and replace this line with `assert msg not in result.stdout_text`

**Comment by amore17** [comment]
Since the issue is not fixed yet.
Test should be added once it get fixed.

**Pull request closed without merging**